client/reader/stream/index.jsx

```diff
@@ -117,7 +119,7 @@ class ReaderStream extends React.Component {
 	}
 
 	_popstate = () => {
-		if ( this.props.selectedPostKey && history.scrollRestoration !== 'manual' ) {
+		if ( this.props.selectedPostKey && window.history.scrollRestoration !== 'manual' ) {
```

Comment:
this one i'm not worried about because it's a handler that would only be called if we had attached to history